core/io/resource_loader.cpp

```diff
@@ -294,6 +294,8 @@ Ref<Resource> ResourceLoader::_load(const String &p_path, const String &p_origin
 	}
 	load_paths_stack.push_back(original_path);
 
+	print_verbose(vformat("Loading resource: %s", p_path));
```

Comment:
Will this spam again to the logs?
Why the p_cache_mode is not respected? An already loaded and cached file should not print the loading message again

Reply:
It may still be somewhat spammy, but not more than it was in the beginning, before I inadvertedly made the change to have the message printed regardless the resource was already cached.

And in that regard we're safe now. If this line is executed, the resource wasn't in cache so it's loaded from disk for sure. The cache mode at this point serves for telling the loader if it has to replace the current cache entry and how to deal with dependencies.

Reply:
Thanks for clarification 👍